### DISCUSSION TITLE
Hello, I'm working on an addon, using @minecraft/server-net.

### DISCUSSION THREAD

**Discussion by sanchezcpp** [JaylyDev/ScriptAPI#125] [Q&A] [answered]
I'm trying to create an addon with access to my external api, I did some tests but no matter what version of the module I put I get "module [Happy Behavior Pack - 1.0.0] depends on unknown module [@minecraft/server-net - 1.0 .0-beta].", if you can help me, thank you in advance.

**Comment by JaylyDev** [comment, marked as answer]
You can only use the server-net module in BDS. If you’re using BDS already follow this [tutorial](https://wiki.bedrock.dev/scripting/script-net.html#setup) to enable server-net.

**Reply by sanchezcpp** [reply]
thanks bro :)
